### ISSUE TITLE
questions about learning rate setting

### ISSUE THREAD

**Issue by strongwolf** [jwyang/faster-rcnn.pytorch#397]
I want to know how do you define the lr for different gpus and batch size?  Which factor controls the lr choice? And one more question, the anchor size is {4,8,16,32} for coco, which corresponds to {64,128,256,512} because of base anchor size 16. Is it suitable for coco as most objects in coco have an area less than 32*32?
